tests/ui/lint/unused/issue-119383.rs

```diff
@@ -0,0 +1,9 @@
+#![feature(if_let_guard)]
```

Comment:
instead of giving this a meaningless issue number as a name, can we give it a descriptive name like "if-let-guard"?